text/0000-buildpack-versions.md

```diff
@@ -0,0 +1,55 @@
+# Meta
+[meta]: #meta
+- **Name:** Semantic Version Formats for Buildpack Versions
+- **Start Date:** 2020-03-11
+- **Supersedes:** N/A
+
+# Summary
+[summary]: #summary
+
+The Buildpacks team wants to enforce format requirements for buildpack versions but will not require version semantics. This RFC explains that buildpack authors are expected to use [semantic version](https://semver.org/) format without enforcing any semantic versioning rules.
+
+# Motivation
+[motivation]: #motivation
+
+There aren't specific guidelines in the buildpack documentation or spec for buildpack authors that explain how versioning works for buildpacks. It should be explicitly stated that the API expects a semantic version format and that authors are not required to follow semantic version semantics.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Buildpack authors are required to use the semantic versioning format, such as `0.10.4`. Currently, the Buildpack API already works with any version format. This requires no additional code to the current code bases, but instead clarification around what the Buildpacks projects expect.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+The Buildpacks team is making a commitment to the future of the project that buildpack developers will have the freedom to use any rules for versioning while using the semantic version format, and this decisions will not break the Buildpack API.
```

Comment:
I think this technically breaks the spec since currently, we [don't specify versioning for buildpacks](https://github.com/buildpacks/spec/blob/master/buildpack.md#buildpacktoml-toml).

As part of this, we should add a [spec change section](https://github.com/buildpacks/rfcs/blob/master/text/0029-template-changes.md).